src/makeRobot.js

```diff
@@ -38,7 +38,82 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return (
+        // eslint-disable-next-line max-len
+        `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`
+      );
+    },
+    get location() {
+      return (`${this.name}: x=${this.coords.x}, y=${this.coords.y}`);
+    },
+    goForward(step = 0) {
```

Comment:
Add new line between methods
```suggestion
    get info() {
      return (
        // eslint-disable-next-line max-len
        `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`
      );
    },
    
    get location() {
      return (`${this.name}: x=${this.coords.x}, y=${this.coords.y}`);
    },
    
    goForward(step = 0) {
```